Content.Shared/Disease/DiseaseEffect.cs

```diff
@@ -15,6 +15,30 @@ public abstract class DiseaseEffect
         /// What effect the disease will have.
         /// </summary>
         public abstract void Effect(DiseaseEffectArgs args);
+
+        /// <summary>
+        /// What is the minimal severity needed for this effect to occur?
+        /// </summary>
+        [DataField("minSeverity")]
+        public float MinSeverity
+        {
+            get => _minSeverity;
+            set => _minSeverity = Math.Clamp(value, 0.0f, 1.0f);
+        }
+
+        private float _minSeverity = 0.0f;
```

Comment:
Why do you need these pairs of public and private floats?

Reply:
Better to just have it fail if someone tries to set it to say, "25" assuming that means 25% so it's clear something is wrong

Reply:
so do I throw an out of bound exception?